src/components/application_manager/src/commands/hmi/navi_start_stream_request.cc

```diff
@@ -110,7 +110,7 @@ void NaviStartStreamRequest::on_event(const event_engine::Event& event) {
       }
       if (hmi_apis::Common_Result::REJECTED == code) {
         LOG4CXX_INFO(logger_, "StartStream response REJECTED ");
-        SendRequest();
+        RetryStartSession();
```

Comment:
Because this is wrong logic and SDL should resend StartStream request in case user pressed "cancel" on pop-up